### ISSUE TITLE
Compile Error on OSX

### ISSUE THREAD

**Issue by rickbutton** [llvm-dcpu16/llvm-dcpu16#156]
A couple of weeks ago, I successfully downloaded this compiler and compiled it on OSX Lion and it worked perfectly fine. I am using a new install and when I try to compile (`make -j4`) it throws this error.

```
/Users/rickbutton/code/dcpu/llvm-dcpu16/tools/clang/include/clang/AST/Expr.h:1392:56: error: no
      member named 'removeZero' in 'llvm::StringRef'
        return StringRef(StrData.asChar, getByteLength()).removeZero();
```

Is this just a syntax error or is something wrong on my end?

**Comment by rickbutton** [comment]
Fixed. It was because the instructions in the README still point to krasin/llvm-dcpu16 rather than llvm-dcpu16/llvm-dcpu16. I suggest changing that reference.

**Comment by rickbutton** [comment]
Fixed with pull request #157
